language-configuration.json

```diff
@@ -75,12 +75,6 @@
             "\""
         ]
     ],
-    "folding": {
-        "markers": {
-            "start": "^\\s*\\b(?:for|parfor|spmd|if|while|try|switch|function|classdef|properties|enumeration|events|methods|arguments)\\b",
-            "end": "\\bend\\b"
```

Comment:
Is there a way to update the "end" rule here to be what we wish (or a close approximation of it)?

Reply:
I don't think this would be easy to achieve. Basically, we would need a regular expression which matches "end", except when found in various contexts. I think this might be possible for several (but not all - e.g. "end" within a block comment), but the complexity may lead to additional poor behavior or edge case bugs.